Catalog revision (#25)
ezsystems/commerce-documentation: docs/guide/catalog/catalog_api/producttype.md

|`addChildProducts`|`ProductNode[]`||InvalidArgumentException|Adds the products passed as argument to the list of child products|
|`hasChildProducts`||int||Returns the number of children|
|`getChildProductBySku`|string|CatalogElement||Returns the child product that has the SKU passed as an argument|
|`addChildProduct`|`ProductNode`|||Adds a single product to the list of child products|

## ProductType search

> A `ProductType` element can appear in the search result of products.

Reviewer: ```suggestion
A `ProductType` element can appear in the result of a search for products.
```

> For this to happen you need to modify the configuration by adding `ses_product_type` to the product list or to product search.

Reviewer: ```suggestion
For this to happen you need to modify the configuration by adding `ses_product_type` to the product list or product search.
```

> `use_display_in_product_list_flag` enables or disables the use of the `productType` flag when displaying in search and in product list.

Reviewer: ```suggestion
`use_display_in_product_list_flag` enables or disables the use of the `productType` flag when displayed in product search and product list.
```

Additionally, a Product Type indexes all data from its products.
You can disable this using the following configuration:

``` yaml
parameters:
    siso_search.default.cp_to_product_type: false
```

> Disabling this lowers indexer times but product data is not indexed alongside Product Types.

Reviewer: ```suggestion
Disabling this feature results in quicker indexing but product data is not indexed alongside Product Types.
```
